osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -16,18 +16,22 @@ namespace osu.Game.Rulesets.Osu.Difficulty.Skills
     public class Speed : OsuStrainSkill
     {
         private const double single_spacing_threshold = 125;
+        private const double rhythm_multiplier = 0.675;
+        private const int history_time_max = 5000; // 5 seconds of calculatingRhythmBonus max.
+        private const double min_speed_bonus = 75; // ~200BPM
+        private const double max_speed_bonus = 45;
```

Comment:
`max_speed_bonus` can be removed since it isn't used anymore.

Reply:
done.